### PR TITLE
Add accelerator type section to `text-embeddings` template

### PR DESCRIPTION
**Pull request by scottjlee**
- Add short text + code to demonstrate specifying a specific accelerator type
- Remove text related to tensor parallelism, since it is confusing for most non-advanced users without documentation (we currently don't have good documentation around this)

### CODE REVIEW

File: templates/text-embeddings/README.ipynb [hunk 1]
```diff
@@ -346,7 +346,13 @@
...
+    "To use GPUs for inference in the Workspace, we can specify `num_gpus` and `concurrency` in the `ds.map_batches()` call below to indicate the number of embedding models and the number of GPUs per model instance, respectively. For example, with `concurrency=4` and `num_gpus=1`, we have 4 embedding model instances, each using 1 GPU, so we need 4 GPUs total.\n",
+    "\n",
+    "You can also specify which accelerator type (for example, if your model requires a specific GPU) to use with the `accelerator_type` parameter. For a full list of supported accelerator types, see [the documentation](https://docs.ray.io/en/latest/ray-core/accelerator-types.html).\n",
+    "\n",
+    "If you do not need to select a particular instance type, you can omit this parameter and select \"Auto-select worker nodes\" under the compute configuration to have Ray select the best worker nodes from the available types:\n",
+    "\n",
+    "![title](assets/ray-data-gpu.png)"
```
Comment (scottsun94) [inline templates/text-embeddings/README.ipynb:355]: Hmm. This image should only highlight the "Auto-select worker nodes" checkbox? We should not highlight the name of the worker node below? I don't know which to look at.

**Comment by scottsun94** [comment]
Left a minor comment. Others LGTM

**Comment by scottjlee** [comment]
@scottsun94 i cropped the image to focus on the auto-select nodes only

**Comment by scottsun94** [comment]
> @scottsun94 i cropped the image to focus on the auto-select nodes only

Currently, I remember that we previously got feedback that they don't know where to find this "auto-select worker nodes" checkbox. Maybe use this screenshot with more UI context?
Screenshot 2024-04-30 at 12 52 39 PM

**Comment by scottjlee** [comment]
> > @scottsun94 i cropped the image to focus on the auto-select nodes only
> 
> I remember that we previously got feedback that they don't know where to find this "auto-select worker nodes" checkbox. Maybe use this screenshot with more UI context? Screenshot 2024-04-30 at 12 52 39 PM

thanks, updated with the new image!

**Review by scottsun94 — APPROVED** [review]

**Pull request merged**